webpack/assets/javascripts/react_app/components/Layout/Layout.js

```diff
@@ -44,6 +44,8 @@ class Layout extends React.Component {
     }
     // changeActive on Back navigation
     window.addEventListener('popstate', () => this.changeActiveOnBack());
+
+    this.props.getTours();
```

Comment:
I haven't found a better place for this API call, I don't think it make sense to make an API on each tour component,  therefore a Root component is required, and the layout answers it atm